resources/assets/lib/beatmap-discussions/love-beatmap-modal.tsx

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+import { BeatmapsetJson } from 'beatmapsets/beatmapset-json';
+import BeatmapJson from 'interfaces/beatmap-json';
+import GameMode from 'interfaces/game-mode';
+import { route } from 'laroute';
+import { computed, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import * as React from 'react';
+import { group as groupBeatmaps } from 'utils/beatmap-helper';
+
+interface Props {
+  beatmapset: BeatmapsetJson;
+  onClose: () => void;
+}
+
+@observer
+export default class LoveConfirmation extends React.Component<Props> {
+  @observable private selectedBeatmapIds: Set<number>;
+
+  constructor(props: Props) {
+    super(props);
+
+    this.selectedBeatmapIds = new Set(this.beatmaps.map((beatmap) => beatmap.id));
+  }
+
+  @computed
+  private get beatmaps() {
+    return this.props.beatmapset.beatmaps?.filter((beatmap) => beatmap.deleted_at === null) ?? [];
+  }
+
+  @computed
+  private get groupedBeatmaps() {
+    return groupBeatmaps(this.beatmaps);
+  }
+
+  render() {
+    return (
+      <div className='love-beatmap-modal'>
+        <div className='love-beatmap-modal__row love-beatmap-modal__row--title'>
+          {osu.trans('beatmaps.nominations.love_choose')}
+        </div>
+
+        <div className='love-beatmap-modal__row love-beatmap-modal__row--content u-fancy-scrollbar'>
```

Comment:
not actually visible in mobile but the scrollbar class need to be on parent for it to be applied correctly

Reply:
I forgot to change this when changed the overflow component in the style.